TZStackView/TZStackView.swift

```diff
@@ -253,6 +262,7 @@ public class TZStackView: UIView {
     
     public func addArrangedSubview(view: UIView) {
         view.translatesAutoresizingMaskIntoConstraints = false
+        view.removeConstraints(view.constraints.filter(constraintIsAutoGenerated))
```

Comment:
Arranged subviews of a stack view should definitely not receive autogenerated constraints.

> Em 28 de fev de 2016, às 12:42, CosynPa <notifications@github.com> escreveu:
> 
> In TZStackView/TZStackView.swift:
> 
> > @@ -253,6 +262,7 @@ public class TZStackView: UIView {
> >      
> >      public func addArrangedSubview(view: UIView) {
> >          view.translatesAutoresizingMaskIntoConstraints = false
> > +        view.removeConstraints(view.constraints.filter(constraintIsAutoGenerated))
> Is this necessary? These constraints should be used for the subviews of view
> 
> —
> Reply to this email directly or view it on GitHub.
>